src/pivotal-ui/components/dropdowns/dropdowns.scss

```diff
@@ -381,6 +397,18 @@ This is a notification dropdown with alerts.
   }
 }
 
+.dropdown-toggle {
+  .caret {
+    display: inline-block;
+    border: 0;
+    margin-right: -0.6em;
+    padding-top: 7px;
+    margin-left: 5px;
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 20px;
```

Comment:
`width` and `height` are proportionate based upon the `icon` class. In the case of the `caret`, here. I'm inflating a bit. I've changed to `1.25em` in the most recent push. Makes it way better for proportional changes.